pkg/sink/sink.go

```diff
@@ -119,6 +129,10 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 			code = thiscode
 			break
 		}
+		if thiscode == http.StatusBadRequest && code != http.StatusCreated {
+			code = http.StatusBadRequest
+			errMsg = marshalErr
+		}
 		if thiscode < code {
 			code = thiscode
```

Comment:
IIUC, this doesn't actually solve the problem, since the next successful trigger that returns 200 will overwrite the code, but keep the error.

Reply:
I changed the code so that we don't get any error in this case.